spec/lib/mailchimp_spec.rb

```diff
@@ -74,13 +77,29 @@
 																body: an_instance_of(String)
 														},
 														{
-																method: 'DELETE',
-																path: 'lists/list_id/members/on_mailchimp'
-														}
+															method: 'DELETE',
+															path: 'lists/list_id/members/on_mailchimp'
+														}								
```

Comment:
Please remove these changes.

Reply:
What changes are you asking to be removed? @wwahammy 
We found that those lines had been removed so I am adding them back.

Reply:
The white space is different from the original, which means the lines are different for Git. It's good practice to avoid having any changes in a file unless they relate to the actual changes being considered.